.changeset/three-llamas-camp.md

```
@@ -0,0 +1,6 @@
+---
+'@mastra/platform-workspace': patch
+'@mastra/railway': patch
+---
+
+Declared checkpoint support (`supportsCheckpoints`) so checkpoint-based features like warm base images and boot-from-checkpoint know snapshots are real.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)